lib/pmdtester/parsers/options.rb

```diff
@@ -32,15 +33,21 @@ def parse(argv)
       Slop.parse argv do |o|
         o.string '-r', '--local-git-repo', 'path to the local PMD repository', required: true
         o.string '-b', '--base-branch', 'name of the base branch in local PMD repository'
-        o.string '-p', '--patch-branch', 'name of the patch branch in local PMD repository', required: true
+        o.string '-p', '--patch-branch',
+                 'name of the patch branch in local PMD repository', required: true
         o.string '-bc', '--base-config', 'path to the base PMD configuration file'
         o.string '-pc', '--patch-config', 'path to the patch PMD configuration file'
         o.string '-c', '--config', 'path to the base and patch PMD configuration file'
-        o.string '-l', '--list-of-project', 'path to the file which contains the list of standard projects', required: true
-        o.string '-m', '--mode', "the mode of the tool: 'local', 'online' or 'single'\n" +
-            "\tsingle: Set this option to 'single' if your patch branch contains changes for any option that can't work on master/base branch\n" +
-            "\tonline: Set this option to 'online' if you want to download the PMD report of master/base branch rather than generating it locally\n" +
-            "\tlocal: Default option is 'local'"
+        o.string '-l', '--list-of-project',
+                 'path to the file which contains the list of standard projects', required: true
+        o.string '-m', '--mode', "the mode of the tool: 'local', 'online' or 'single'\n" \
+                                 "\tsingle: Set this option to 'single' if your patch branch " \
```

Comment:
Hm.... this now isn't so readable as it was before. Can you try to use a [Here Document](http://ruby-doc.org/core-2.5.1/doc/syntax/literals_rdoc.html#label-Here+Documents)? I'm not sure, whether it would directly work as an argument or whether you would need to define a local variable before, but a here document might help in improving the readability of the help text for option "--mode".